packages/beacon-node/test/e2e/chain/bls/multithread.test.ts

```diff
@@ -78,6 +78,7 @@ describe("chain / bls / multithread queue", function () {
         expect(isValid).to.deep.equal([true, true, true], `sig set ${i} returned invalid`);
       }
     }
+    await pool.close();
```

Comment:
how is this related to to CI time reduction?  will this stop the hanging we see by chance?